gnucash/report/standard-reports/account-piecharts.scm

```diff
@@ -570,8 +570,8 @@ balance at a given time"))
                                     (qof-print-date to-date)))
                        (if show-total?
                            (let ((total (apply + (unzip1 combined))))
-                             (format
-                              #f ": ~a"
+                             (string-append
+                              ": "
```

Comment:
In both cases I am not sure, how the result will look in LTR languages like ar or he. Can you test it with LANG=he_IL.utf8 or an ar variant?